src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
True.

My thinking was that we only allow `Vector`, `Point` and `Normal` from the UI anyway and I remembered you saying that we want to keep the serialisation minimal - so I added `Color` in that conditional to remove it from the serialisation of the `Color3fPlug` if I remember correctly.